html/selionGrid.html

```diff
@@ -110,6 +98,26 @@ <h2 id="start-selion-node">Start the Grid Node</h2>
   <div class="container">
     <img src="../images/grid-management-console.jpg" alt="View of the SeLion Grid Management Console">
   </div>
+
+  <h2 id="selion-config">SeLion Grid Configuration File</h2>
+  <p>
+    The SeLion Grid Configuration JSON file allows Grid to load the default properties. File available in
+    <code>$SELION_HOME/config/SeLionConfig.json</code>. Default JSON file will be overrided by <code>-selionConfig</code> argument.
+  </p>
+  <ol>
+    <li>restartCycle - Restart cycle will check specified interval to start the server if it is down. Units in milliseconds.</li>
+    <li>uniqueSessionCount – Number of unique session completion node will be recycled.</li>
```

Comment:
Should say, `Number of unique sessions before automatically recycling the node. Default: 50`